spec/spec_helper.rb

```diff
@@ -1,3 +1,9 @@
-require 'coveralls'
+require 'coveralls'  
 Coveralls.wear!
+
+# couldn't get to work, error: Nokogiri was built against LibXML version 2.9.3, but has dynamically loaded 2.9.2  T
+# tried to fix with 'bundle exec gem pristine nokogiri' but just wiped out nokogiri.  http://codebeerstartups.com/2013/07/warning-nokogiri-was-built-against-libxml-version-2-9-0-but-has-dynamically-loaded-2-7-8/ 
```

Comment:
Metrics/LineLength: Line is too long. [212/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)